### ISSUE TITLE
[Question] Node execution way

### ISSUE THREAD

**Issue by shawwillen** [miroiu/nodify#143]
I would like to ask: in the calculator example, the execution of a node is based on the change of its port value. However, is it possible to implement the same execution way as visual scripts in Unity or UE4 blueprints?

**Comment by miroiu** [comment]
Yes, it is possible. Nodify is a controls library to help you build custom node-based editors, so it doesn’t enforce a specific approach to app logic. For example, take a look at the StateMachine example (specifically the `StateMachine.Start` method). This example demonstrates another way of traversing and executing nodes.

**Comment by shawwillen** [comment]
Thank you, I will try something similar later.

**Comment by miroiu** [comment]
Set `IsAnimatingDirectionalArrows` to `true` on the connection. I recommend you check out the Playground app, which demonstrates most of the features.

**Comment by miroiu** [comment]
Please reopen the issue if you need more information.
